lib/notification-issue.coffee

```diff
@@ -52,7 +54,10 @@ class NotificationIssue
       "#{repoUrl}/issues/new?title=#{@encodeURI(@getIssueTitle())}&body=#{@encodeURI(issueBody)}"
 
   getIssueTitle: ->
-    @notification.getMessage()
+    title = @notification.getMessage()
+    if title.length > TRUNCATE_TITLE_TO
```

Comment:
It looks like checking the length isn't necessary:

```javascript
foo = '123'    // => "123"
foo = foo.substring(0, 10) + '...'     // => "123..."
```

So 58-60 can just be compressed into:

```coffee
title.substring(0, TRUNCATE_TITLE_TO - 3) + '...'
```